certs.sh

```diff
@@ -13,7 +13,7 @@ openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 \
 
 echo "Generating CSR"
 openssl req -out $CERTDIR/cert.csr -new -newkey rsa:2048 -nodes -keyout $CERTDIR/priv.key \
-  -subj "/O=interop runner/"
+  -subj "/O=interop runner/CN=interoprunner/"
```

Comment:
This is inconsistent with the `subjectAltName`, isn't it?

Reply:
oh good catch. 
Discarding this change anyway. We are fixing it in Fizz